### DISCUSSION TITLE
Storage Offering for Clients

### DISCUSSION THREAD

**Discussion by jsim77** [filecoin-project/community#493] [Market]
Hi, I'm a newbie and have set up Filecoin into my Mac Terminal. Well, my intention is for my business to offer business people to store their files securely. Under the FileCoin, those clients I closed the deal with, for this storage, are they able to get Filecoin token? What about myself for offering the storage service, am I eligible to get the Filecoin token for each successful storage closed dealing? How to increase the storage size for my clients as time goes by? How much is the pricing per storage size? Appreciate your advice, many thanks.

**Comment by s3w47m88** [comment]
I'm with @jsim77

My background is website development, web application development in PHP, JS, HTML/CSS on Laravel and other platforms that have an ecosystem.

I'm reading all of the documentation but what's still abstract is an understanding of how a storage provider gets paid.

As a web developer I'm thinking "I setup a physical server in my office, install Lotus, follow documentation on how to create a Sector, and then at some point in the configuration process add a wallet ID or something for Filecoin to be deposited into.

But I'm literally Day 1 on this, so my perspective may be totally abstract to how it really functions.
